ci/pipeline.yml

```diff
@@ -0,0 +1,152 @@
+---
+groups:
+  - name: bosh-oracle-cpi-release
+    jobs:
+      - test-unit
+      - build-candidate
+      - setup-infrastructure
+      - deploy-ubuntu
```

Comment:
Need a job to run the functional tests `make ocitests`

Reply:
yes

Reply:
At what stage do you want to run `make ocitest` - in between `test-unit` and `build-candidate`?

Reply:
Yes, that sounds reasonable.  We don't need to build a cpi-release tarball for running ocitest.